server/src/main/java/org/opensearch/action/search/SearchPhaseResults.java

```diff
@@ -75,7 +75,7 @@ final int getNumShards() {
 
     void consumeShardFailure(int shardIndex) {}
 
-    AtomicArray<Result> getAtomicArray() {
+    public AtomicArray<Result> getAtomicArray() {
```

Comment:
Since this is now public, we should probably add JavaDoc.

Also, as a pet peeve, I never like methods named by the type of thing they return, rather than the conceptual thing returned. Like, this method called `getAtomicArray` returns an `AtomicArray`, but what's the nature of that array? What's it for?

Reply:
I will add javadoc. 

The atomic array holds the results from the shards. `Result` is what is being returned from shards